ipinfo/handler_utils.py

```diff
@@ -52,12 +54,13 @@ def get_headers(access_token):
     return headers
 
 
-def format_details(details, countries, eu_countries):
+def format_details(details, countries, eu_countries, countries_flags):
     """
     Format details given a countries object.
     """
     details["country_name"] = countries.get(details.get("country"))
     details["isEU"] = details.get("country") in eu_countries
+    details["country_flag"] = countries_flags.get(details.get("country"))
```

Comment:
This is subtle, but note that this gives you the same object as is found in the class - it won't be deep-copied. That means if the user modifies this field, it will affect all lookups in the future too. It's a shared object.

We may need to do a python deep-copy on this object. https://docs.python.org/3/library/copy.html